app/src/main/java/io/appium/uiautomator2/handler/GetElementAttribute.java

```diff
@@ -56,7 +79,90 @@ public AppiumResponse safeHandle(IHttpRequest request) {
         } catch (UiAutomator2Exception e) {
             Logger.error(MessageFormat.format("Unable to retrive attribute {0}", attributeName), e);
             return new AppiumResponse(getSessionId(request), WDStatus.UNKNOWN_ERROR, e);
+        } catch (ReflectiveOperationException | InvalidSelectorException e) {
+            Logger.error("Can not access to method or field: ", e);
+            return new AppiumResponse(getSessionId(request), WDStatus.UNKNOWN_ERROR);
```

Comment:
would it be useful to return some error text or explanation here as in the previous catches?